src/Microsoft.AspNetCore.Mvc.Analyzers/DiagnosticDescriptors.cs

```diff
@@ -15,5 +15,32 @@ public static class DiagnosticDescriptors
                 "Usage",
                 DiagnosticSeverity.Warning,
                 isEnabledByDefault: true);
+
+        public static readonly DiagnosticDescriptor MVC1001_FiltersShouldNotBeAppliedToPageHandlerMethods =
+            new DiagnosticDescriptor(
+                "MVC1001",
+                "Filters cannot be applied to page handler methods.",
+                "'{0}' cannoted be applied to a Razor Page handler methods. It may be applied either to a PageModel or globally.",
```

Comment:
typo: 'cannot' and remove 'a'

Reply:
'It may be applied either to a PageModel or globally.' => 'It may be applied either to a PageModel class or globally.'?